apps/qubit/modules/actor/templates/_occupations.php

```diff
@@ -11,7 +11,7 @@
   </thead><tbody>
 
     <?php $i = 0; foreach ($occupations as $item) { ?>
-      <?php $form->getWidgetSchema()->setNameFormat("occupations[$i][%s]"); ?>
+      <?php $form->getWidgetSchema()->setNameFormat("occupations[${i}][%s]"); ?>
```

Comment:
These are normalized later to `{$i}`, like the method/property calls (`{$request->query}` above).